azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/tools/common/util/CommandUtil.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.tools.common.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.commons.io.IOUtils;
+
+import com.microsoft.azure.tools.common.exception.CommandExecuteException;
+
+public class CommandUtil {
+    private static final boolean isWindows = System.getProperty("os.name").contains("Windows");
+
+    public static String executeCommandAndGetOutput(final String cmd, File cwd)
+            throws IOException, InterruptedException {
+        final String[] cmds = new String[]{isWindows ? "cmd.exe" : "bash", "/c", cmd};
```

Comment:
should be "-c" for none windows system?

Reply:
fixed in https://github.com/microsoft/azure-maven-plugins/pull/1205